src/app.jsx

```diff
@@ -31,11 +31,9 @@ class Application extends React.Component {
         super(props);
         this.state = {
             version: { version: "unknown" },
-            images: [],
-            containers: [],
-            imagesMeta: [],
-            containersMeta: [],
-            containersStats:[],
+            images: [], /* images[Id]: detail info of image with Id from InspectImage */
+            containers: [], /* containers[Id] detail info of container with Id from InspectContainer */
+            containersStats:[], /* containersStats[Id] memory usage of running container with Id */
```

Comment:
These should be objects then (`{}`), not arrays. Otherwise it's all too easy to iterate over them with `map()` or similar, and wondering why nothing comes out of it.